src/Files.App/Views/ColumnShellPage.xaml.cs

```diff
@@ -230,7 +230,7 @@ private async void ColumnShellPage_PreviewKeyDown(object sender, KeyRoutedEventA
 
 			switch (c: ctrl, s: shift, a: alt, t: tabInstance, k: args.Key)
 			{
-				case (true, false, false, true, (VirtualKey)192): // ctrl + ` (accent key), open terminal
+				case (true, _, false, true, (VirtualKey)192): // ctrl + ` (accent key), open terminal
```

Comment:
Shouldn't it say "true" instead of "_"?

Reply:
I think the way this should be done is with another case where Shift is 'true' and then there pass the "-Verb runas" as an argument.

Reply:
I don't think so, it would only create duplicate code

Reply:
Alright, I don't know if a following commit trampled over these changes, but the feature for "CTRL+SHIFT+`" is not working. 
#11443

Reply:
I'll check

Reply:
> I'll check

I did, but unsure if the feature was moved elsewhere. On commit:
https://github.com/files-community/Files/commit/dc676d30a6e01836683e9bde4bed88ac92f65be5

Reply:
Moved elsewhere but still it should work. I'll see I can find what is causing the problem

Reply:
Can you confirm that it was working before that commit?

Reply:
Ok I found the root and I feel so stupid ...

Reply:
> Ok I found the root and I feel so stupid ...

You're smart enough to have found it! 
Great work 😄

Reply:
I wouldn't have without your help